### ISSUE TITLE
When using PEP-526 to specify the type of a class variable and then attempting to docstring it, I get "String statement has no effect (pointless-string-statement)"

### ISSUE THREAD

**Issue by terrisgit** [PyCQA/pylint#2066]
This issue was first discussed at https://stackoverflow.com/questions/49737835/pep-526-is-not-compatible-with-docstrings-for-class-variables . I realize this is a gray area in Python docstrings but this warning is a barrier (for my project at least) to fully embracing PEP-526. As with #193 I would like pylint to ignore this use of docstrings.

### joe.py
```
"""Module docstring"""
class Joe(object):
    """Class docstring"""
    counter: int
    """Data docstring"""
```

### pylint x.py
```
************* Module joe
W:  5, 4: String statement has no effect (pointless-string-statement)

---------------------------------------------------------------------
Your code has been rated at 6.67/10
```

### pylint --version output
```
Using config file .pylintrc
pylint 1.8.4, 
astroid 1.6.3
Python 3.6.5 (default, Apr 20 2018, 10:13:23) 
[GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.1)]
```

**Comment by PCManticore** [comment]
Thanks for the issue! pylint is not yet aware of PEP 526.
